facebook/OpenBIC
# Artemis: cb: Latch USB reset pin to avoid unexpected reset USB after …

…BIC warm reset

# Description
- Latch USB reset/USB mux pin to avoid unexpected resetting USB after BIC warm reset.
- Latch ACCL power good pin to avoid unexpected resetting ACCL power good pin after BIC warm reset

# Motivation
- Due to hardware design issues, it is necessary to avoid resetting USB reset/USB mux/ACCL power good pin during the BIC reset process.

# Test Plan
- Build code: Pass
- Check GPIO value after BIC warm reset: Pass

# Log
uart:~$ platform gpio list_all
[0  ] SMB_PMBUS_ALERT_N_R                : PP  | input (I) | 1(1)
[1  ] INT_SMB_BIC_PEX0_N_R               : PP  | input (I) | 1(1)
[2  ] INT_SMB_BIC_PEX1_N_R               : PP  | input (I) | 1(1)
[3  ] SMB_P1V25_ALRT_N_R                 : PP  | input (I) | 1(1)
[4  ] INA233_ACCL1_ALRT_N_R              : PP  | input (I) | 1(1)
[5  ] INA233_ACCL2_ALRT_N_R              : PP  | input (I) | 1(1)
[6  ] INA233_ACCL3_ALRT_N_R              : PP  | input (I) | 1(1)
[7  ] FIO_PWRBTN_N_R                     : PP  | input (I) | 1(1)
[8  ] ASD_MUX2_SEL                       : PP  | output(O) | 1(1)
[9  ] ASD_MUX2_EN_R4_N                   : PP  | input (I) | 0(0)
[10 ] INA233_ACCL4_ALRT_N_R              : PP  | input (I) | 1(1)
[11 ] INA233_ACCL5_ALRT_N_R              : PP  | input (I) | 1(1)
[12 ] INA233_ACCL6_ALRT_N_R              : PP  | input (I) | 1(1)
[13 ] TYPE_C_PLUG_DETECT_R               : PP  | input (I) | 1(1)
[14 ] MEB_STRAP0_N_R                     : PP  | input (I) | 0(0)
[15 ] MEB_STRAP1_N_R                     : PP  | input (I) | 0(0)
[16 ] RST_SMB_MUX0_ACCL_N                : OD  | output(I) | 1(1)
[17 ] RST_SMB_MUX1_ACCL_N                : OD  | output(I) | 1(1)
[18 ] RST_BMC_FROM_ACB_N                 : PP  | output(O) | 1(1)
[19 ] INA233_ACCL7_ALRT_N_R              : PP  | input (I) | 1(1)
[20 ] SPI_ROM0_SEL                       : PP  | output(O) | 1(1)
[21 ] SPI_ROM1_SEL                       : PP  | output(O) | 1(1)
[22 ] ACB_THERM_OVERT_N                  : OD  | output(I) | 1(1)
[23 ] RST_BIC_SELF_HW_RST_N              : PP  | output(O) | 1(1)
[24 ] RST_USB_HUB0_N                     : OD  | output(I) | 1(1)
[25 ] RST_USB_HUB1_N                     : PP  | output(I) | 0(0)
[26 ] RST_USB_HUB2_N                     : OD  | output(I) | 1(1)
[27 ] RST_USB_HUB3_N                     : PP  | output(I) | 0(0)
[28 ] RST_USB_HUB4_N                     : OD  | output(I) | 1(1)
[29 ] RST_USB_HUB5_N                     : OD  | output(I) | 1(1)
[32 ] FIO_REV_ID0_R                      : PP  | input (I) | 0(0)
[33 ] FIO_REV_ID1_R                      : PP  | input (I) | 1(1)
[34 ] FIO_REV_ID2_R                      : PP  | input (I) | 1(1)
[35 ] PRSNT_FIO_N                        : PP  | input (I) | 0(0)
[36 ] HS1_UV_WARN_N                      : PP  | input (I) | 1(1)
[37 ] HS1_OC_WARN_N                      : PP  | input (I) | 1(1)
[38 ] HS2_UV_WARN_N                      : PP  | input (I) | 1(1)
[39 ] HS2_OC_WARN_N                      : PP  | input (I) | 1(1)
[40 ] SMB_P0V8_ALERT_N                   : PP  | input (I) | 1(1)
[41 ] HS1_FAULT_N                        : PP  | input (I) | 1(1)
[42 ] HS2_FAULT_N                        : PP  | input (I) | 1(1)
[43 ] ACB_BIC_READY_N                    : OD  | output(O) | 0(0)
[44 ] USB2_MUX0_SEL                      : OD  | output(I) | 1(1)
[45 ] USB2_MUX1_SEL                      : OD  | output(I) | 1(1)
[46 ] USB2_MUX3_SEL                      : OD  | output(O) | 0(0)
[47 ] EN_JTAG_BIC_CPLD_N                 : PP  | output(O) | 1(1)
[48 ] INT_SMB_MEB_0_N_R                  : PP  | input (I) | 0(0)
[49 ] INT_SMB_MEB_1_N_R                  : PP  | input (I) | 0(0)
[50 ] INT_U228_N                         : PP  | input (I) | 1(1)
[51 ] INT_U229_N                         : PP  | input (I) | 1(1)
[52 ] INT_U230_N                         : PP  | input (I) | 1(1)
[53 ] INT_U233_N                         : PP  | input (I) | 0(0)
[54 ] INT_POWER_STATUS_N_R               : PP  | input (I) | 1(1)
[55 ] EN_5V_EFUSE                        : PP  | output(O) | 1(1)
[88 ] P3V3_ACCL12_PG                     : PP  | input (I) | 1(1)
[89 ] P3V3_ACCL11_PG                     : PP  | input (I) | 1(1)
[90 ] P3V3_ACCL10_PG                     : PP  | input (I) | 1(1)
[91 ] P3V3_ACCL9_PG                      : PP  | input (I) | 1(1)
[92 ] P3V3_ACCL8_PG                      : PP  | input (I) | 1(1)
[93 ] P3V3_ACCL7_PG                      : PP  | input (I) | 1(1)
[94 ] P3V3_ACCL6_PG                      : PP  | input (I) | 1(1)
[95 ] P3V3_ACCL5_PG                      : PP  | input (I) | 1(1)
[97 ] P3V3_ACCL4_PG                      : PP  | input (I) | 1(1)
[99 ] P3V3_ACCL3_PG                      : PP  | input (I) | 1(1)
[100] P3V3_ACCL2_PG                      : PP  | input (I) | 1(1)
[101] P3V3_ACCL1_PG                      : PP  | input (I) | 1(1)
[108] BOARD_ID1                          : PP  | input (I) | 1(1)
[109] BOARD_ID2                          : PP  | input (I) | 1(1)
[110] BOARD_ID3                          : PP  | input (I) | 1(1)
[112] BOARD_ID0                          : PP  | input (I) | 1(1)
[113] REV_ID0                            : PP  | input (I) | 0(0)
[114] REV_ID1                            : PP  | input (I) | 0(0)
[115] REV_ID2                            : PP  | input (I) | 1(1)
[127] INA233_ACCL10_ALRT_N_R             : PP  | input (I) | 1(1)
[131] INA233_ACCL11_ALRT_N_R             : PP  | input (I) | 1(1)
[132] INA233_ACCL12_ALRT_N_R             : PP  | input (I) | 1(1)
[145] INA233_ACCL8_ALRT_N_R              : PP  | input (I) | 1(1)
[146] INA233_ACCL9_ALRT_N_R              : PP  | input (I) | 1(1)
